docs/docs/eslint.md

```diff
@@ -33,57 +33,14 @@ touch .eslintrc.js
 
 ### Configuring ESLint
 
-We recommend copying our default .eslintrc.js content below to your newly created `.eslintrc.js` file and modifying it per your needs. Reference ESLint's [rules documentation](https://eslint.org/docs/rules/) for more options.
+We recommend copying the following .eslintrc.js content below to your newly created `.eslintrc.js` file and modifying it per your needs. Reference ESLint's [rules documentation](https://eslint.org/docs/rules/) for more options.
```

Comment:
Add a note that this mimics the default eslint config that Gatsby ships with.

Reply:
Might be worth mentioning and linking the shipped ESLint file here too for reference. So that developers are aware that a more complex version exists internally: https://github.com/gatsbyjs/gatsby/blob/master/.eslintrc.json